Client/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -352,7 +352,7 @@ class ShieldsViewController: UIViewController, PopoverContentComponent {
     case fingerprintingProtection
   }
   
-  private func recordShieldsUpdateP3A(shield: BraveShield) {
+  static func recordShieldsUpdateP3A(shield: BraveShield) {
```

Comment:
Do you think it might be a better idea to create a Manager class etc and inject this dependency in place where 3pa for record functions is a better idea or keeping them in their respective places is actually makes more sense ?

Reply:
Manager class is probably a bit heavy since we'd need different ones for many different questions and they'd all be mostly single function classes but we could take that direction in some form. I'm going to move this out of `ShieldsViewController` though just to get rid of this static method